dapp/src/theme/utils/zIndices.ts

```diff
@@ -1,5 +1,9 @@
 export const zIndices = {
-  sidebar: 1450,
+  sidebar: 1400,
```

Comment:
I just noticed this but the buttons on the sidebar don't work. This is probably because we should not change the `z-index` for the sidebar. I think we just need to back to the previous value? 🤔


https://github.com/user-attachments/assets/3f4d841d-9d35-4cf9-9aad-6f32e31fabf9

Reply:
There is an issue with stacking contexts here. Currently Sidebar has lower stacking context so it remains "under" the modal - not clickable. The `1400` value of `z-index` might work here if modal and sidebar nodes were in the same root node. Reverting the value to `1450` resolves the issue but doesn't fix the issue with header. Now the header is inoperable when the modal is open eg. you can't disconnect wallet. If we bring the header's stacking context higher, the modal close button will not work when scrolled and so on, and so on...
There are way too many stacking contexts which got clamped and I'm afraid it's impossible to pass it through without changing the DOM structure. I played around with various `z-index` values over elements and fixing one issue only brings another. 

The desired solution would be to render the Sidebar component alongside with the Modal - in the same portal node. It will cause the Sidebar to inherit the stacking context of the modal so it will be operable without the need of enforcing it manually with `z-index` property.

As a temporary solution I'd revert the change to `1450` keeping in mind now the Header is inoperable so it needs to be fixed as well. I'd suggest to leave a comment in code and handle this fix in separate PR. The priority of changing the account with open modal is lower than the ability of closing modal on smaller screens. You can change the account elsewhere, however with inoperable Modal's close button you get stuck.

https://github.com/user-attachments/assets/6436b4a9-21b2-4ab9-880f-eba57d62f06b

Reply:
The temporary fix works. I added a ticket for the proper fix to the backlog https://github.com/thesis/acre/issues/832.